doc/api_2_0_spec/admin_api_2_0_spec.md

```diff
@@ -213,112 +207,6 @@ Output:
 { "success": 1 }
 ```
 
-## api/v2/stories/cliff
-
```

Comment:
These too!

I don't have enough brain capacity to understand Git's manpages so I typically use one of the Git's GUI tools for that. When I was using a Mac, my favorite was [Sourcetree](https://www.sourcetreeapp.com/) (free!) but then my beloved Mac almost caught fire so I resorted to [Sublime Merge](https://www.sublimemerge.com/) (paid, but doesn't beg for $ too often).

Fastest way to recover these accidentally removed blocks with a GUI tool would be to find out the commit that got them removed, search / scroll down to that commit on a GUI tool, select the lines that you want to restore, click on "Revert" or something like that, and commit the change once you're happy with it. You can do it [using the terminal too](https://stackoverflow.com/a/5669428/200603).